azure-servicebus/src/test/java/com/microsoft/azure/servicebus/TestUtils.java

```diff
@@ -10,6 +10,11 @@
 	
 	private static final String NAMESPACE_CONNECTION_STRING_ENVIRONMENT_VARIABLE_NAME = "AZURE_SERVICEBUS_JAVA_CLIENT_TEST_CONNECTION_STRING";
     public static final String FIRST_SUBSCRIPTION_NAME = "subscription1";
+
+    private static final Boolean RUN_WITH_PROXY = false;
```

Comment:
Please let me know if there is a cleaner way to do this.

Reply:
could be an environment variable name like connection string. so that things can be changed via command line without compiling the code.

---
In reply to: [218255335](https://github.com/Azure/azure-service-bus-java/pull/279#discussion_r218255335) [](ancestors = 218255335)

Reply:
Yes. Read an environment variable, along with proxy host name and port.

---
In reply to: [218598117](https://github.com/Azure/azure-service-bus-java/pull/279#discussion_r218598117) [](ancestors = 218598117,218255335)

Reply:
I have added a commit that should resolve this